pylint/checkers/refactoring.py

```diff
@@ -261,13 +266,42 @@ def _check_redefined_argument_from_local(self, name_node):
                                  args=(name_node.name, ))
 
     @utils.check_messages('redefined-argument-from-local',
-                          'too-many-nested-blocks')
+                          'too-many-nested-blocks',
+                          'mutable-sequence-modified-in-loop')
     def visit_for(self, node):
         self._check_nested_blocks(node)
 
         for name in node.target.nodes_of_class(astroid.AssignName):
             self._check_redefined_argument_from_local(name)
 
+        if self._check_iter_is_infer_list(node):
+            for part in node.nodes_of_class(astroid.Expr):
+                if self._check_part_is_call_method_elimination(node, part):
+                    self.add_message('mutable-sequence-modified-in-loop',
+                                     node=part,
+                                     args=(node.iter.name))
+
+    @staticmethod
+    def _check_iter_is_infer_list(node):
+        """ Check if node.iter is astroid.util.Uninferable o
```

Comment:
These docstrings have an unusual format. They should be formed out of a short summary, empty line, followed by a more verbose description.  And please don't indent them like this, follow PEP 257 instead.